src/olympia/activity/tests/test_utils.py

```diff
@@ -517,6 +517,36 @@ def test_staff_cc_group_get_mail(self, send_mail_mock):
                           review_url,
                           'you are member of the activity email cc group.')
 
+    @mock.patch('olympia.activity.utils.send_mail')
+    def test_staff_cc_group_needinfo_correct_subject(self, send_mail_mock):
```

Comment:
can you check in here that the developer subject line is still the same - so you're checking both subjects.